docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -203,184 +203,150 @@ files.
-    installation.
+	```bash
+	$ sudo lvcreate -L 90G -n data vg-docker
+	Logical volume `data` created.
+	```
 
-    ```bash
-    $ rm -rf /var/lib/docker/*
-    ```
+	The command creates an LVM logical volume called `data` and an associated
+	block device file at `/dev/vg-docker/data`. In a later step, you instruct the
+	`devicemapper` storage driver to use this block device to store image and
+	container data.
 
-14. Configure the Engine daemon with specific devicemapper options.
+	If you receive a signature detection warning, make sure you are working on
+	the correct devices before continuing. Signature warnings indicate that the
+	device you're working on is currently in use by LVM or has been used by LVM in
+	the past.
 
-    There are two ways to do this. You can set options on the commmand line if you start the daemon there:
+6. Create a new logical volume (LV) called `metadata` from space in the
+`vg-docker` volume group.
 
-    ```bash
-    --storage-driver=devicemapper --storage-opt=dm.thinpooldev=/dev/mapper/docker-thinpool --storage-opt dm.use_deferred_removal=true
-    ```
+	```bash
+	$ sudo lvcreate -L 4G -n metadata vg-docker
+	Logical volume `metadata` created.
+	```
 
-    You can also set them for startup in the `daemon.json` configuration, for example:
+	This creates an LVM logical volume called `metadata` and an associated
+	block device file at `/dev/vg-docker/metadata`. In the next step you instruct
+	the `devicemapper` storage driver to use this block device to store image and
+	container metadata.
 
-    ```json
-     {
-             "storage-driver": "devicemapper",
-             "storage-opts": [
-                     "dm.thinpooldev=/dev/mapper/docker-thinpool",
-                     "dm.use_deferred_removal=true"
-             ]
-     }
-    ```
-15. Start the Engine daemon.
+7. Start the Docker daemon with the `devicemapper` storage driver and the
+`--storage-opt` flags.
 
-    ```bash
-    $ systemctl start docker
-    ```
+	The `data` and `metadata` devices that you pass to the `--storage-opt`
+	options were created in the previous steps.
 
-After you start the Engine daemon, ensure you monitor your thin pool and volume
-group free space. While the volume group will auto-extend, it can still fill
-up. To monitor logical volumes, use `lvs` without options or `lvs -a` to see tha
-data and metadata sizes. To monitor volume group free space, use the `vgs` command.
+	```bash
+	$ sudo docker daemon --storage-driver=devicemapper --storage-opt dm.datadev=/dev/vg-docker/data --storage-opt dm.metadatadev=/dev/vg-docker/metadata &
+	[1] 2163
+	[root@ip-10-0-0-75 centos]# INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
+	INFO[0027] Option DefaultDriver: bridge
+	INFO[0027] Option DefaultNetwork: bridge
+	<-- output truncated -->
+	INFO[0027] Daemon has completed initialization
+	INFO[0027] Docker daemon commit=1b09a95-unsupported graphdriver=aufs version=1.11.0-dev
```

Comment:
Can you remove the `-unsupported` here? It probably because you built this daemon yourself, but not needed here in the docs